update.sh

```diff
@@ -25,12 +26,16 @@ curl -sSL "${packagesUrl}" > "$packages"
 for version in "${versions[@]}"; do
 	fullVersion="$(sed 's/;/;\n/g' $packages | grep -e 'php-'"$version"'.*\.tar\.bz2' | sed -r 's/.*php-('"$version"'[^"]+)\.tar\.bz2.*/\1/' | sort -V | tail -1)"
 	gpgKey="${gpgKeys[$version]}"
-	if [ -z "$gpgKey" ]; then
-		echo >&2 "ERROR: missing GPG key fingerprint for $version"
-		echo >&2 "  try looking on http://php.net/downloads.php#gpg-$version"
-		exit 1
+	if [ -z "$fullVersion" ]; then
```

Comment:
This logic was changed a bit to skip with a warning versions that don't exist on the remote URL in https://github.com/docker-library/php/pull/110/files#diff-25e6ad8ebc5d26ba06d2bcb0b220149c.